### PR TITLE
Wrong instruction in building from source for Windows #7912

### PR DESCRIPTION
**Pull request by nonconvexopt**
Modified trivial errors in Google JAX docs about building from source instruction in Windows.
[#7912](https://github.com/google/jax/issues/7912)

### CODE REVIEW

File: docs/developer.md [hunk 1]
```diff
@@ -109,6 +110,7 @@ python .\build\build.py `
   --cuda_version='10.1' `
   --cudnn_version='7.6.5'
 ```
+Discard `--cuda_compute_capabilities` flag in case of argument exception.
```
Comment (hawkinsp) [inline docs/developer.md:113]: I'd remove this comment. But instead I'd just remove `--cuda_compute_capabilities` from the command above. The defaults should work for most people, and even if they do not, it isn't a Windows-specific thing.

(It turns out we also accidentally removed the flag, but we'll add it back again.)

File: docs/developer.md [hunk 2]
```diff
@@ -92,8 +92,9 @@ is needed. See [Installing Bazel on Windows](https://docs.bazel.build/versions/m
 for more details. Install the following packages:
 
 ```
-pacman -S patch realpath
+pacman -S patch coreutils
 ```
+`realpath` command should be found in shell command line.
```
Comment (hawkinsp) [inline docs/developer.md:97]: Nit: perhaps:
"Once `coreutils` is installed, the `realpath` command should be present in your shell's path."

Or maybe just omit this? If we are installing the right package above, won't all the right things happen already?

**Review by hawkinsp — CHANGES_REQUESTED** [review]

File: docs/developer.md (hunk 1 above)
Comment (nonconvexopt) [reply docs/developer.md]: That would be more straightforward. Thank you for your comment.

File: docs/developer.md (hunk 2 above)
Comment (nonconvexopt) [reply docs/developer.md]: I revised it according to your comment.

**Review by hawkinsp — APPROVED** [review]

**Pull request merged**